modelon/impact/client/entities/workspace.py

```diff
@@ -0,0 +1,545 @@
+
+    def get_fmu(self, fmu_id: str) -> ModelExecutable:
+        """
+        Returns a ModelExecutable class object.
+
+        Returns:
+
+            FMU --
+                ModelExecutable class object.
+
+        Example::
+
+            workspace.get_fmu(fmu_id)
+        """
+        resp = self._workspace_sal.fmu_get(self._workspace_id, fmu_id)
+        return ModelExecutable(
+            self._workspace_id,
+            resp["id"],
+            self._workspace_sal,
+            self._model_exe_sal,
+            resp,
+        )
+
+    def get_experiments(self) -> List[Experiment]:
+        """
+        Returns a list of Experiment class objects.
+
+        Returns:
+
+            experiment --
+                List of Experiment class objects.
+
+        Example::
+
+            workspace.get_experiments()
+        """
+        resp = self._workspace_sal.experiments_get(self._workspace_id)
+        return [
+            Experiment(
+                self._workspace_id,
+                item["id"],
+                self._workspace_sal,
+                self._model_exe_sal,
+                self._exp_sal,
+                item,
+            )
+            for item in resp["data"]["items"]
+        ]
+
+    def get_experiment(self, experiment_id: str) -> Experiment:
+        """
+        Returns an Experiment class object.
+
+        Parameters:
+
+            experiment_id --
+                The ID of the experiment.
+
+        Returns:
+
+            experiment --
+                Experiment class object.
+
+        Example::
+
+            workspace.get_experiment(experiment_id)
+        """
+        resp = self._workspace_sal.experiment_get(self._workspace_id, experiment_id)
+        return Experiment(
+            self._workspace_id,
+            resp["id"],
+            self._workspace_sal,
+            self._model_exe_sal,
+            self._exp_sal,
+            resp,
+        )
+
+    def create_experiment(
+        self,
+        definition: ExperimentDefinition,
+        user_data: Optional[Dict[str, Any]] = None,
+    ) -> Experiment:
+        """Creates an experiment.
+        Returns an Experiment class object.
+
+        Parameters:
+
+            definition --
+                An parametrized experiment definition class of type
+                modelon.impact.client.experiment_definition.SimpleModelicaExperimentDefinition
+                or
+                modelon.impact.client.experiment_definition.SimpleFMUExperimentDefinition.
+            user_data --
+                Optional dictionary object with custom data to attach to the experiment.
+
+        Returns:
+
+            experiment --
+                Experiment class object.
+
+        Example::
+
+            workspace.create_experiment(definition)
+        """
+        if isinstance(
+            definition,
+            (SimpleFMUExperimentDefinition, SimpleModelicaExperimentDefinition),
+        ):
+            experiment_definition = definition.to_dict()
```

Comment:
experiment_definition -> definition_as_dict would better describe what is happening here